python/desc/skycatalogs/catalog_creator.py

```diff
@@ -795,3 +803,18 @@ def write_config(self, overwrite=False, path_only=False):
 
         self._written_config = config.write_config(self._config_path,
                                                    overwrite=overwrite)
+
+    def write_provenance_file(self, datafile_path):
+        '''
+        Write git provenance to a yaml file with name derived from a
+        just-written datafile name
+        '''
+        dir_name, base_name = os.path.split(datafile_path)
+        base_parts = base_name.split('.')
+        base_parts[-1] = 'yaml'
+        base_parts[-2] = base_parts[-2] + '_provenance'
+        new_base = '.'.join(base_parts)
+        outpath = os.path.join(dir_name, new_base)
```

Comment:
I think you can accomplish the same result with
```
outpath = datafile_path.rsplit('.', 1)[0] + '_provenance.yaml'
```

Reply:
That certainly is much simpler! Thanks.